src/Routing/Filter/ThrottleFilter.php

```diff
@@ -35,20 +35,35 @@ public function beforeDispatch(Event $event)
         $this->_initCache();
         $this->_count = $this->_touch();
 
+        $config = $this->getConfig();
+
         // client has not exceeded rate limit
-        if ($this->_count <= $this->getConfig('limit')) {
-            $this->_setHeaders($event->getData('response'));
+        if ($this->_count <= $config['limit']) {
+            $this->_setHeaders($event->getData['response']);
```

Comment:
`getData()` is a method not array :)

Reply:
:rofl: This why we don't have to use GitHub, sorry :joy: